extensions/wikia/PhalanxII/models/PhalanxContentModel.php

```diff
@@ -59,14 +58,12 @@ public function getBlockMessage() {
 	}
 
 	public function contentBlock() {
-		$msg = "{$this->block->text} (Block #{$this->block->id})";
-		$this->logBlock();
+		$msg = "{$this->block->getText()} (Block #{$this->block->getId()})";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$msg". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)

[project-id]: wikia:app